src/apps/setup.ts

```diff
@@ -1,6 +1,7 @@
+import { IncomingMessage, ServerResponse } from "http";
```

Comment:
FYI we had issues with Deno with these types in Octokit.

It's worth investigating if it is still the case

Reply:
I somehow think, that this is never reported to probot. E.g. in src/server.ts we import HttpServer which should have also resulted in some regressions.